lib/scenario-editor/utils/valhalla.js

```diff
@@ -257,7 +257,7 @@ export function routeWithGraphHopper (points: Array<LatLng>, avoidMotorways?: bo
 
   const params = {
     key: graphHopperKey,
-    vehicle: 'car',
+    profile: 'car',
     debug: true,
```

Comment:
For backwards compatibility, should we include both?

Reply:
It's not possible to include both. You get a exception as response. I'm thinking of adding a parameter to add to the config file, that's set the response for version 7. Default for 5 a 6.